patches/0003-YYUSE.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/parse.y b/parse.y
+index 0fd0c46..97c9743 100644
+--- a/parse.y
++++ b/parse.y
+@@ -12673,7 +12673,6 @@ count_char(const char *str, int c)
+ RUBY_FUNC_EXPORTED size_t
+ rb_yytnamerr(struct parser_params *p, char *yyres, const char *yystr)
+ {
+-    YYUSE(p);
```

Comment:
Patch to Remove YYUSE (which is now private in bison 3.7.5+)

cf https://github.com/ruby/ruby/commit/b091889ed67f278b1652ddde88596e2160d5aef0#diff-60e6f0aa6e190c54f6cb0ac74148ced3cf0ef8703e1c3824a07a82dc54654953R13335